src/main/kotlin/com/criptext/mail/scenes/settings/cloudbackup/CloudBackupActivity.kt

```diff
@@ -82,6 +88,40 @@ class CloudBackupActivity: BaseActivity(){
         return controller
     }
 
+    private fun setNewAttachmentsAsActivityMessage(data: Intent?, filePickerConst: String?) {
```

Comment:
This function appears in 3 classes. what if you create a util function that returns the attachments list?